policies/recipes/steps.go

```diff
@@ -15,17 +15,27 @@
 package recipes
 
 import (
+	"archive/tar"
+	"archive/zip"
+	"compress/bzip2"
+	"compress/gzip"
 	"fmt"
 	"io"
+	"io/ioutil"
 	"os"
 	"os/exec"
 	"path/filepath"
 	"runtime"
 	"strconv"
+	"strings"
+	"time"
 
 	"github.com/GoogleCloudPlatform/osconfig/common"
 
 	osconfigpb "github.com/GoogleCloudPlatform/osconfig/_internal/gapi-cloud-osconfig-go/google.golang.org/genproto/googleapis/cloud/osconfig/v1alpha2"
+
```

Comment:
you have some extra lines between imports, have all std imports together, then one line then all thrid_party imports

Reply:
Done.